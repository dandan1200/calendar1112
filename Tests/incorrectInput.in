python calendar.py ADD 10-10-10 
python calendar.py ADD 10-10-10000 "Test"
python calendar.py ADD
python calendar.py GET
python calendar.py GET 10-10-2000
python calendar.py UPD 10-10-2000 "NO"
