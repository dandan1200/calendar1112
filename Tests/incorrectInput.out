Missing event name
Multiple errors occur
Unable to parse date
Error: not enough arguments
Error: not enough arguments
Not enough arguments given
Unable to update, event does not exist
